### PR TITLE
feat: add msServices mfa calls

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

ref: MANAGER-3014

ff443d3 — feat: add msServices mfa calls

### CODE REVIEW

File: src/api/msServices/msServices.service.v6.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ angular
       edit: {
         method: 'PUT', cache, isArray: false, interceptor,
       },
+      createMfaOnAllUsers: {
+        method: 'POST',
+        url: '/msServices/:serviceName/createMfaOnAllUsers ',
```
Comment (antleblanc) [inline src/api/msServices/msServices.service.v6.js]: `interception` isn't needed in your case?

File: src/api/msServices/account/mfa/mfa.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+angular.module('ovh-api-services').service('OvhApiMsServicesAccountMfaV6', ($resource, $cacheFactory) => {
+  const cache = $cacheFactory('OvhApiMsServicesAccountMfaV6');
```
Comment (antleblanc) [inline src/api/msServices/account/mfa/mfa.v6.service.js]: Since you've not implemented an `interceptor`, is `$cacheFactory` is needed in your case?

Comment (cbourgois) [reply src/api/msServices/account/mfa/mfa.v6.service.js]: No, removed.

File: src/api/msServices/msServices.service.v6.js (hunk 1 above)
Comment (cbourgois) [reply src/api/msServices/msServices.service.v6.js]: No, because `createMfaOnAllUsers` won't affect the `service`, it create `mfa` on sub-`account`s

**Pull request merged**